matlab/schemas/+stimulus/+conf/Varma.m

```diff
@@ -2,38 +2,123 @@
 control = stimulus.getControl;
 control.clearAll()   % clear trial queue and cached conditions.
 
+paramsM.fps             = 60;
+paramsM.duration        = 180;  % to be changed to 1800s
+paramsM.rng_seed        = 1;
+paramsM.pattern_width   = 32;
+paramsM.pattern_aspect  = 1.7;
+paramsM.pattern_upscale = 10;
+paramsM.ori_coherence   = 2.5;
```

Comment:
The Monet2 parameters have changes.  Please update as in stimulus.conf.matisse2.